.docs/README.md

```diff
@@ -276,25 +257,20 @@ try {
 }
 ```
 
-### Po platbě
+### After payment
 
-Váš zákazník provede potřebné úkony na Gopay platební bráně, a jakmile je proces
-dokončen, je přesměrován zpátky do vaší aplikace, buď na `successUrl`
-nebo `failureUrl` adresu. Obě dvě dostanou od Gopay následující sadu parametrů:
+Your customer will perform the necessary actions on the Gopay payment gateway, and once the process is done completed, is redirected back to your application, either to `successUrl` or `failureUrl` address. Both will receive the following set of parameters from Gopay:
 
 - paymentSessionId
 - targetGoId
-- orderNumber // variabilní číslo
+- orderNumber // varyingg number 
 - encryptedSignature
 
-*Plus parametry, které uvedete v successUrl, resp. failureUrl.*
+*PPlus the parameters you specify in successUrl or failureUrl.*
 
-První parametr je totožný s tím, který jsme si v předchozí kapitole uložili do
-naší interní modelové reprezentace objednávky. Můžeme jej tedy použít k jejímu
-opětovnému načtení.
+The first parameter is identical to the one we saved in the previous chapter our internal model representation of the order. So we can use it for hers reload.
 
-Všechny tyto údaje + údaje z načtené objednávky pak použijeme ke znovusestavení
-objektu platby:
+We will then use all this data + data from the loaded order for reassembly payment object:
```

Comment:
We will then use all this data + data from the loaded order for payment object reassembly: